db/migrate/20240210054714_devise_create_usuarios.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
-class DeviseCreateUsuarios < ActiveRecord::Migration[7.1]
+class DeviseCreateUsers < ActiveRecord::Migration[7.1]
```

Comment:
For old migrations is not necessary did this change. So revert changes for this file and below.

Reply:
Just the content right?

Reply:
No, the entire file should be as it was.

Reply:
ok!